### PR TITLE
Stdlib date handlers

### PR DESCRIPTION
**Pull request by RamilGaripov**

### CODE REVIEW

File: libraries/standard.js [hunk 1]
```diff
@@ -81,6 +81,52 @@ const remove_if_empty = function (object, field) {
...
+  if (Object.keys(item.dates).length == 0) {
+      delete item.dates;
+  }
+  return item;
+}
+
+const remove_if_zero = function(object) {
+  for (const prop in object) {
+      if (object[prop] == 0) delete object[prop];
+      else if (object[prop] == "") delete object[prop];
```
Comment (sarahsemkow) [inline libraries/standard.js:127]: should this check for null and undefined too? if so, you can do:
else if (!object[prop]) delete object[props];

**Review by sarahsemkow — CHANGES_REQUESTED** [review]

File: libraries/standard.js (hunk 1 above)
Comment (mksample) [reply libraries/standard.js:127]: I think 0 in this case is different than undefined or null. If we want to remove undefined/null values we can use remove_if_null but my assumption is that this is intended to work with the dates object which sets fields to 0 or "" by default.

Comment (mksample) [reply libraries/standard.js:127]: plus we don't export it in the std_lib map so I think its safe to leave as is

**Review by mksample — APPROVED** [review]
lgtm!

**Review by sarahsemkow — APPROVED** [review]

**Pull request merged**